ios/Classes/SwiftAddToWalletPlugin.swift

```diff
@@ -34,17 +37,22 @@ class PKAddPassButtonNativeView: NSObject, FlutterPlatformView {
     private var _pass: FlutterStandardTypedData
     private var _width: CGFloat
     private var _height: CGFloat
+    private var _key: String
+    private var _channel: FlutterMethodChannel
 
     init(
         frame: CGRect,
         viewIdentifier viewId: Int64,
         arguments args: [String: Any],
-        binaryMessenger messenger: FlutterBinaryMessenger?
+        binaryMessenger messenger: FlutterBinaryMessenger?,
+        channel: FlutterMethodChannel
     ) {
         _view = UIView()
         _pass = args["pass"] as! FlutterStandardTypedData
         _width = args["width"] as? CGFloat ?? 140
         _height = args["height"] as? CGFloat ?? 30
+        _key = args["key"] as! String
+        _channel = channel
```

Comment:
Passing the entire channel down to the View is a bit hardcore but it's not worth much more effort yet 😅